### ISSUE TITLE
react-scripts build exception

### ISSUE THREAD

**Issue by alexandr2110pro** [easybread/easybread#30]
Example app reports a problem after introducing the DataMapper and `NotString<Person>`  and refactoring.

It takes an enormous amount of time to build the app and the build command reports this:

```console
easybread on  example-app [✘!+?] via ⬢ v12.16.1
➜ yarn build --stream --scope example-ts-express-react
yarn run v1.22.4
$ ./node_modules/.bin/lerna run build --stream --scope example-ts-express-react
lerna notice cli v3.20.2
lerna notice filter including "example-ts-express-react"
lerna info filter [ 'example-ts-express-react' ]
lerna info Executing command in 1 package: "yarn run build"
example-ts-express-react: $ CI=false react-scripts build
example-ts-express-react: Creating an optimized production build...
Cexample-ts-express-react: <--- Last few GCs --->
example-ts-express-react: [2890:0x10291b000]  1804430 ms: Mark-sweep 2040.4 (2054.7) -> 2036.4 (2054.4) MB, 1096.4 / 0.0 ms  (average mu = 0.126, current mu = 0.039) allocation failure scavenge might not succeed
example-ts-express-react: [2890:0x10291b000]  1804439 ms: Scavenge 2040.4 (2054.4) -> 2036.5 (2054.4) MB, 4.2 / 0.0 ms  (average mu = 0.126, current mu = 0.039) allocation failure
example-ts-express-react: [2890:0x10291b000]  1804445 ms: Scavenge 2040.4 (2054.4) -> 2036.4 (2054.7) MB, 4.3 / 0.0 ms  (average mu = 0.126, current mu = 0.039) allocation failure
example-ts-express-react: <--- JS stacktrace --->
example-ts-express-react: ==== JS stack trace =========================================
example-ts-express-react:     0: ExitFrame [pc: 0x100950919]
example-ts-express-react: Security context: 0x1ac0f71c08d1 <JSObject>
example-ts-express-react:     1: trySymbolTable(aka trySymbolTable) [0x1ac017ac1a59] [/Users/acherednichenko/projects/easybread/node_modules/typescript/lib/typescript.js:~37385] [pc=0x15da216b590b](this=0x1ac0ef8c04b1 <undefined>,0x1ac048d0f5e9 <Map map = 0x1ac0e4980989>,0x1ac0ef8c04b1 <undefined>)
example-ts-express-react:     2: getAccessibleSymbolChainFromSymbolTable(aka getAccessibleSymbolChainFromSymbolTabl...
example-ts-express-react: FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
example-ts-express-react: Writing Node.js report to file: report.20200501.081028.2890.0.001.json
example-ts-express-react: Node.js report completed
example-ts-express-react:  1: 0x100080c68 node::Abort() [/usr/local/bin/node]
example-ts-express-react:  2: 0x100080dec node::errors::TryCatchScope::~TryCatchScope() [/usr/local/bin/node]
example-ts-express-react:  3: 0x100185167 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/usr/local/bin/node]
example-ts-express-react:  4: 0x100185103 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/usr/local/bin/node]
example-ts-express-react:  5: 0x10030b2f5 v8::internal::Heap::FatalProcessOutOfMemory(char const*) [/usr/local/bin/node]
example-ts-express-react:  6: 0x10030c9c4 v8::internal::Heap::RecomputeLimits(v8::internal::GarbageCollector) [/usr/local/bin/node]
example-ts-express-react:  7: 0x100309837 v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/usr/local/bin/node]
example-ts-express-react:  8: 0x1003077fd v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/usr/local/bin/node]
example-ts-express-react:  9: 0x100312fba v8::internal::Heap::AllocateRawWithLightRetry(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/usr/local/bin/node]
example-ts-express-react: 10: 0x100313041 v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/usr/local/bin/node]
example-ts-express-react: 11: 0x1002e035b v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType, v8::internal::AllocationOrigin) [/usr/local/bin/node]
example-ts-express-react: 12: 0x100618718 v8::internal::Runtime_AllocateInYoungGeneration(int, unsigned long*, v8::internal::Isolate*) [/usr/local/bin/node]
example-ts-express-react: 13: 0x100950919 Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_NoBuiltinExit [/usr/local/bin/node]
example-ts-express-react: 14: 0x15da216b590b
example-ts-express-react: 15: 0x15da216b27e3
lerna success run Ran npm script 'build' in 1 package in 1823.6s:
lerna success - example-ts-express-react
✨  Done in 1824.79s.
```

**Comment by alexandr2110pro** [comment]
probably adding `--max_old_space_size=4096` can fix the problem. Will try next

**Comment by alexandr2110pro** [comment]
Managed to get report

```json
{
  "header": {
    "reportVersion": 2,
    "event": "Allocation failed - JavaScript heap out of memory",
    "trigger": "FatalError",
    "filename": "report.20200501.100557.5281.0.001.json",
    "dumpEventTime": "2020-05-01T10:05:57Z",
    "dumpEventTimeStamp": "1588316757345",
    "processId": 5281,
    "threadId": null,
    "cwd": "/Users/acherednichenko/projects/easybread/packages/example-ts-express-react",
    "commandLine": [
      "/usr/local/bin/node",
      "--max-old-space-size=2048",
      "/Users/acherednichenko/projects/easybread/packages/example-ts-express-react/node_modules/fork-ts-checker-webpack-plugin/lib/service.js"
    ],
    "nodejsVersion": "v12.16.3",
    "wordSize": 64,
    "arch": "x64",
    "platform": "darwin",
    "componentVersions": {
      "node": "12.16.3",
      "v8": "7.8.279.23-node.35",
      "uv": "1.34.2",
      "zlib": "1.2.11",
      "brotli": "1.0.7",
      "ares": "1.16.0",
      "modules": "72",
      "nghttp2": "1.40.0",
      "napi": "5",
      "llhttp": "2.0.4",
      "http_parser": "2.9.3",
      "openssl": "1.1.1g",
      "cldr": "36.0",
      "icu": "65.1",
      "tz": "2019c",
      "unicode": "12.1"
    },
    "release": {
      "name": "node",
      "lts": "Erbium",
      "headersUrl": "https://nodejs.org/download/release/v12.16.3/node-v12.16.3-headers.tar.gz",
      "sourceUrl": "https://nodejs.org/download/release/v12.16.3/node-v12.16.3.tar.gz"
    },
    "osName": "Darwin",
    "osRelease": "19.4.0",
    "osVersion": "Darwin Kernel Version 19.4.0: Wed Mar  4 22:28:40 PST 2020; root:xnu-6153.101.6~15/RELEASE_X86_64",
    "osMachine": "x86_64",
    "cpus": [
      {
        "model": "Intel(R) Core(TM) i5-4570R CPU @ 2.70GHz",
        "speed": 2700,
        "user": 1765780,
        "nice": 0,
        "sys": 2416040,
        "idle": 8301140,
        "irq": 0
      },
      {
        "model": "Intel(R) Core(TM) i5-4570R CPU @ 2.70GHz",
        "speed": 2700,
        "user": 1807490,
        "nice": 0,
        "sys": 565480,
        "idle": 10109000,
        "irq": 0
      },
      {
        "model": "Intel(R) Core(TM) i5-4570R CPU @ 2.70GHz",
        "speed": 2700,
        "user": 1753200,
        "nice": 0,
        "sys": 441920,
        "idle": 10286690,
        "irq": 0
      },
      {
        "model": "Intel(R) Core(TM) i5-4570R CPU @ 2.70GHz",
        "speed": 2700,
        "user": 1601110,
        "nice": 0,
        "sys": 434220,
        "idle": 10446340,
        "irq": 0
      }
    ],
    "networkInterfaces": [
      {
        "name": "lo0",
        "internal": true,
        "mac": "00:00:00:00:00:00",
        "address": "127.0.0.1",
        "netmask": "255.0.0.0",
        "family": "IPv4"
      },
      {
        "name": "lo0",
        "internal": true,
        "mac": "00:00:00:00:00:00",
        "address": "::1",
        "netmask": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff",
        "family": "IPv6",
        "scopeid": 0
      },
      {
        "name": "lo0",
        "internal": true,
        "mac": "00:00:00:00:00:00",
        "address": "fe80::1",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 1
      },
      {
        "name": "en1",
        "internal": false,
        "mac": "88:63:df:c3:c2:25",
        "address": "fe80::c33:2dda:128f:32f",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 5
      },
      {
        "name": "en1",
        "internal": false,
        "mac": "88:63:df:c3:c2:25",
        "address": "192.168.0.10",
        "netmask": "255.255.255.0",
        "family": "IPv4"
      },
      {
        "name": "awdl0",
        "internal": false,
        "mac": "ce:f8:9e:b2:a5:e0",
        "address": "fe80::ccf8:9eff:feb2:a5e0",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 7
      },
      {
        "name": "llw0",
        "internal": false,
        "mac": "ce:f8:9e:b2:a5:e0",
        "address": "fe80::ccf8:9eff:feb2:a5e0",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 8
      },
      {
        "name": "utun0",
        "internal": false,
        "mac": "00:00:00:00:00:00",
        "address": "fe80::7efc:1a55:22d4:c11a",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 12
      },
      {
        "name": "utun1",
        "internal": false,
        "mac": "00:00:00:00:00:00",
        "address": "fe80::b6bd:8775:c560:9a85",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 13
      },
      {
        "name": "utun2",
        "internal": false,
        "mac": "00:00:00:00:00:00",
        "address": "fe80::ad1:fb38:1799:c34d",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 14
      },
      {
        "name": "utun3",
        "internal": false,
        "mac": "00:00:00:00:00:00",
        "address": "fe80::5d17:d39f:e3c7:89d2",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 15
      }
    ],
    "host": "As-iMac.local"
  },
  "javascriptStack": {
    "message": "No stack.",
    "stack": ["Unavailable."]
  },
  "nativeStack": [
    {
      "pc": "0x0000000100164e03",
      "symbol": "report::TriggerNodeReport(v8::Isolate*, node::Environment*, char const*, char const*, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, v8::Local<v8::String>) [/usr/local/bin/node]"
    },
    {
      "pc": "0x000000010008645a",
      "symbol": "node::OnFatalError(char const*, char const*) [/usr/local/bin/node]"
    },
    {
      "pc": "0x0000000100187c07",
      "symbol": "v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/usr/local/bin/node]"
    },
    {
      "pc": "0x0000000100187ba7",
      "symbol": "v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/usr/local/bin/node]"
    },
    {
      "pc": "0x0000000100315955",
      "symbol": "v8::internal::Heap::FatalProcessOutOfMemory(char const*) [/usr/local/bin/node]"
    },
    {
      "pc": "0x00000001003171ca",
      "symbol": "v8::internal::Heap::RecomputeLimits(v8::internal::GarbageCollector) [/usr/local/bin/node]"
    },
    {
      "pc": "0x0000000100313bfc",
      "symbol": "v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/usr/local/bin/node]"
    },
    {
      "pc": "0x00000001003119fe",
      "symbol": "v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/usr/local/bin/node]"
    },
    {
      "pc": "0x000000010031d8ca",
      "symbol": "v8::internal::Heap::AllocateRawWithLightRetry(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/usr/local/bin/node]"
    },
    {
      "pc": "0x000000010031d951",
      "symbol": "v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/usr/local/bin/node]"
    },
    {
      "pc": "0x00000001002eb58a",
      "symbol": "v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType, v8::internal::AllocationOrigin) [/usr/local/bin/node]"
    },
    {
      "pc": "0x000000010063e698",
      "symbol": "v8::internal::Runtime_AllocateInYoungGeneration(int, unsigned long*, v8::internal::Isolate*) [/usr/local/bin/node]"
    },
    {
      "pc": "0x000000010097cc39",
      "symbol": "Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_NoBuiltinExit [/usr/local/bin/node]"
    }
  ],
  "javascriptHeap": {
    "totalMemory": 2162606080,
    "totalCommittedMemory": 2161328648,
    "usedMemory": 2115890184,
    "availableMemory": 61673440,
    "memoryLimit": 2197815296,
    "heapSpaces": {
      "read_only_space": {
        "memorySize": 262144,
        "committedMemory": 33088,
        "capacity": 32808,
        "used": 32808,
        "available": 0
      },
      "new_space": {
        "memorySize": 25165824,
        "committedMemory": 25165824,
        "capacity": 12569472,
        "used": 4848,
        "available": 12564624
      },
      "old_space": {
        "memorySize": 2089689088,
        "committedMemory": 2088987136,
        "capacity": 2070519872,
        "used": 2069189744,
        "available": 1330128
      },
      "code_space": {
        "memorySize": 2265088,
        "committedMemory": 2162656,
        "capacity": 1983840,
        "used": 1983840,
        "available": 0
      },
      "map_space": {
        "memorySize": 1052672,
        "committedMemory": 808680,
        "capacity": 645360,
        "used": 645360,
        "available": 0
      },
      "large_object_space": {
        "memorySize": 44122112,
        "committedMemory": 44122112,
        "capacity": 44030800,
        "used": 44030800,
        "available": 0
      },
      "code_large_object_space": {
        "memorySize": 49152,
        "committedMemory": 49152,
        "capacity": 2784,
        "used": 2784,
        "available": 0
      },
      "new_large_object_space": {
        "memorySize": 0,
        "committedMemory": 0,
        "capacity": 12569472,
        "used": 0,
        "available": 12569472
      }
    }
  },
  "resourceUsage": {
    "userCpuSeconds": 1953.51,
    "kernelCpuSeconds": 40.3147,
    "cpuConsumptionPercent": 112.519,
    "maxRss": 2271693766656,
    "pageFaults": {
      "IORequired": 1,
      "IONotRequired": 1350382
    },
    "fsActivity": {
      "reads": 0,
      "writes": 0
    }
  },
  "libuv": [],
  "workers": [],
  "environmentVariables": {
    
  },
  "userLimits": {
    "core_file_size_blocks": {
      "soft": 0,
      "hard": "unlimited"
    },
    "data_seg_size_kbytes": {
      "soft": "unlimited",
      "hard": "unlimited"
    },
    "file_size_blocks": {
      "soft": "unlimited",
      "hard": "unlimited"
    },
    "max_locked_memory_bytes": {
      "soft": "unlimited",
      "hard": "unlimited"
    },
    "max_memory_size_kbytes": {
      "soft": "unlimited",
      "hard": "unlimited"
    },
    "open_files": {
      "soft": 10240,
      "hard": "unlimited"
    },
    "stack_size_bytes": {
      "soft": 8388608,
      "hard": 67104768
    },
    "cpu_time_seconds": {
      "soft": "unlimited",
      "hard": "unlimited"
    },
    "max_user_processes": {
      "soft": 1392,
      "hard": 1392
    },
    "virtual_memory_kbytes": {
      "soft": "unlimited",
      "hard": "unlimited"
    }
  },
  "sharedObjects": [
    "/usr/local/bin/node",
    "/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation",
    "/usr/lib/libSystem.B.dylib",
    "/usr/lib/libc++.1.dylib",
    "/usr/lib/libobjc.A.dylib",
    "/usr/lib/libfakelink.dylib",
    "/usr/lib/libDiagnosticMessagesClient.dylib",
    "/usr/lib/libicucore.A.dylib",
    "/usr/lib/libz.1.dylib",
    "/usr/lib/libc++abi.dylib",
    "/usr/lib/system/libcache.dylib",
    "/usr/lib/system/libcommonCrypto.dylib",
    "/usr/lib/system/libcompiler_rt.dylib",
    "/usr/lib/system/libcopyfile.dylib",
    "/usr/lib/system/libcorecrypto.dylib",
    "/usr/lib/system/libdispatch.dylib",
    "/usr/lib/system/libdyld.dylib",
    "/usr/lib/system/libkeymgr.dylib",
    "/usr/lib/system/liblaunch.dylib",
    "/usr/lib/system/libmacho.dylib",
    "/usr/lib/system/libquarantine.dylib",
    "/usr/lib/system/libremovefile.dylib",
    "/usr/lib/system/libsystem_asl.dylib",
    "/usr/lib/system/libsystem_blocks.dylib",
    "/usr/lib/system/libsystem_c.dylib",
    "/usr/lib/system/libsystem_configuration.dylib",
    "/usr/lib/system/libsystem_coreservices.dylib",
    "/usr/lib/system/libsystem_darwin.dylib",
    "/usr/lib/system/libsystem_dnssd.dylib",
    "/usr/lib/system/libsystem_featureflags.dylib",
    "/usr/lib/system/libsystem_info.dylib",
    "/usr/lib/system/libsystem_m.dylib",
    "/usr/lib/system/libsystem_malloc.dylib",
    "/usr/lib/system/libsystem_networkextension.dylib",
    "/usr/lib/system/libsystem_notify.dylib",
    "/usr/lib/system/libsystem_sandbox.dylib",
    "/usr/lib/system/libsystem_secinit.dylib",
    "/usr/lib/system/libsystem_kernel.dylib",
    "/usr/lib/system/libsystem_platform.dylib",
    "/usr/lib/system/libsystem_pthread.dylib",
    "/usr/lib/system/libsystem_symptoms.dylib",
    "/usr/lib/system/libsystem_trace.dylib",
    "/usr/lib/system/libunwind.dylib",
    "/usr/lib/system/libxpc.dylib",
    "/Users/acherednichenko/projects/easybread/packages/example-ts-express-react/node_modules/fork-ts-checker-webpack-plugin/node_modules/fsevents/fsevents.node",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices",
    "/System/Library/Frameworks/CFNetwork.framework/Versions/A/CFNetwork",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/FSEvents.framework/Versions/A/FSEvents",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/CarbonCore.framework/Versions/A/CarbonCore",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/Metadata.framework/Versions/A/Metadata",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/OSServices.framework/Versions/A/OSServices",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/SearchKit.framework/Versions/A/SearchKit",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/AE.framework/Versions/A/AE",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/LaunchServices",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/DictionaryServices.framework/Versions/A/DictionaryServices",
    "/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/SharedFileList.framework/Versions/A/SharedFileList",
    "/System/Library/Frameworks/Foundation.framework/Versions/C/Foundation",
    "/System/Library/Frameworks/IOKit.framework/Versions/A/IOKit",
    "/System/Library/Frameworks/SystemConfiguration.framework/Versions/A/SystemConfiguration",
    "/System/Library/Frameworks/Security.framework/Versions/A/Security",
    "/usr/lib/libsqlite3.dylib",
    "/usr/lib/libxml2.2.dylib",
    "/usr/lib/libnetwork.dylib",
    "/usr/lib/libapple_nghttp2.dylib",
    "/usr/lib/libauto.dylib",
    "/usr/lib/libcompression.dylib",
    "/System/Library/Frameworks/DiskArbitration.framework/Versions/A/DiskArbitration",
    "/usr/lib/libarchive.2.dylib",
    "/usr/lib/liblangid.dylib",
    "/usr/lib/libCRFSuite.dylib",
    "/usr/lib/liblzma.5.dylib",
    "/usr/lib/libenergytrace.dylib",
    "/usr/lib/libbsm.0.dylib",
    "/usr/lib/system/libkxld.dylib",
    "/System/Library/PrivateFrameworks/AppleFSCompression.framework/Versions/A/AppleFSCompression",
    "/usr/lib/libcoretls.dylib",
    "/usr/lib/libcoretls_cfhelpers.dylib",
    "/usr/lib/libpam.2.dylib",
    "/usr/lib/libxar.1.dylib",
    "/usr/lib/libbz2.1.0.dylib",
    "/usr/lib/libiconv.2.dylib",
    "/usr/lib/libcharset.1.dylib",
    "/usr/lib/libpcap.A.dylib",
    "/System/Library/Frameworks/NetFS.framework/Versions/A/NetFS",
    "/System/Library/PrivateFrameworks/NetAuth.framework/Versions/A/NetAuth",
    "/System/Library/PrivateFrameworks/login.framework/Versions/A/Frameworks/loginsupport.framework/Versions/A/loginsupport",
    "/System/Library/PrivateFrameworks/TCC.framework/Versions/A/TCC",
    "/System/Library/PrivateFrameworks/CoreNLP.framework/Versions/A/CoreNLP",
    "/System/Library/PrivateFrameworks/MetadataUtilities.framework/Versions/A/MetadataUtilities",
    "/usr/lib/libmecabra.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate",
    "/usr/lib/libmecab.dylib",
    "/usr/lib/libgermantok.dylib",
    "/usr/lib/libThaiTokenizer.dylib",
    "/usr/lib/libChineseTokenizer.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vImage.framework/Versions/A/vImage",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/vecLib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libvMisc.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libvDSP.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLAPACK.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLinearAlgebra.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libSparseBLAS.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libQuadrature.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBNNS.dylib",
    "/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libSparse.dylib",
    "/System/Library/PrivateFrameworks/LanguageModeling.framework/Versions/A/LanguageModeling",
    "/System/Library/PrivateFrameworks/CoreEmoji.framework/Versions/A/CoreEmoji",
    "/System/Library/PrivateFrameworks/LinguisticData.framework/Versions/A/LinguisticData",
    "/System/Library/PrivateFrameworks/Lexicon.framework/Versions/A/Lexicon",
    "/usr/lib/libcmph.dylib",
    "/System/Library/Frameworks/OpenDirectory.framework/Versions/A/Frameworks/CFOpenDirectory.framework/Versions/A/CFOpenDirectory",
    "/System/Library/Frameworks/OpenDirectory.framework/Versions/A/OpenDirectory",
    "/System/Library/PrivateFrameworks/APFS.framework/Versions/A/APFS",
    "/System/Library/Frameworks/SecurityFoundation.framework/Versions/A/SecurityFoundation",
    "/usr/lib/libutil.dylib",
    "/System/Library/PrivateFrameworks/CoreServicesStore.framework/Versions/A/CoreServicesStore",
    "/System/Library/Frameworks/ServiceManagement.framework/Versions/A/ServiceManagement",
    "/System/Library/PrivateFrameworks/BackgroundTaskManagement.framework/Versions/A/BackgroundTaskManagement",
    "/usr/lib/libxslt.1.dylib"
  ]
}
```

**Comment by alexandr2110pro** [comment]
Managed to launch the builder with extended memory limit (forcing it to run forked processes like `fork-ts-checker-webpack-plugin` to use extended memory too) with react-app-rewired.

It still doesn't help.

**Comment by alexandr2110pro** [comment]
Ended up creating a package with custom schema interfaces. schema-dts seems to be completely screwed up at least now.

**Comment by wreiske** [comment]
Possibly related to https://github.com/google/schema-dts/issues/34

**Comment by Eyas** [comment]
Note that we got some advice from TS team here: microsoft/Typescript#37214

tl;dr: TypeScript 3.9 makes this much better w/ schema-dts. But increasing heap size should work.

Schema.org is just big, and not easy to prune, but we're looking into it.

**Comment by Eyas** [comment]
Is this still an issue, given the resolution to #13?

If so, we _think_ we greatly reduced type checking overhead in schema-dts v0.8.1

**Comment by alexandr2110pro** [comment]
Hey @Eyas 

We'll check but for now, we've made a decision to move forward with maintaining our own "schemas" because schema.org definitions seem not to work well for many real-life cases. Think of access token response etc.

But we're trying to keep our definition as similar to schema.org as possible and probably someday we'll find a use for your library too.

So generally, no it is not an issue now.

**Comment by alexandr2110pro** [comment]
Closing due to #13 and https://github.com/easybread/easybread/issues/30#issuecomment-734448848
